app/Domains/Users/Policies/UserPolicy.php

```diff
@@ -14,7 +14,7 @@ class UserPolicy
     /**
      * Determine whether the authenticated user can delete an other user or not.
      *
-     * @param  User $user The resource entity from the authenticated user.
+     * @param User $user The resource entity from the authenticated user.
```

Comment:
Hey, 

Can u align the docblock with one extra space. 

Example: 

**now:** 

```
/**
  * Determine whether the authenticated user can delete an other user or not.
  *
  * @param  User $user The resource entity from the authenticated user.
  * @param User $user The resource entity from the authenticated user.
  * @return bool
  */
```

**what i mean:** 

```
/**
  * Determine whether the authenticated user can delete an other user or not.
  *
  * @param  User $user The resource entity from the authenticated user.
  * @param  User $user The resource entity from the authenticated user.
  * @return bool
  */
```